emukit/model_wrappers/gpy_model_wrappers.py

```diff
@@ -24,12 +24,13 @@ def __init__(self, gpy_model: GPy.core.Model, n_restarts: int = 1):
         self.model = gpy_model
         self.n_restarts = n_restarts
 
-    def predict(self, X: np.ndarray) -> Tuple[np.ndarray, np.ndarray]:
+    def predict(self, X: np.ndarray, full_cov: bool = False) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
That's bad design in GPy, because it changes the structure of the output. Can you instead define a separate method that would internally call `self.model.predict(X, full_cov=True)`?

Reply:
Fixed now